github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -176,6 +176,69 @@ presubmits:
           path: /var/log/audit
           type: Directory
         name: audit
+  - always_run: false
+    run_before_merge: true
+    annotations:
+      fork-per-release: "true"
+      testgrid-dashboards: kubevirt-presubmits
+    cluster: kubevirt-prow-workloads
+    decorate: true
+    decoration_config:
+      grace_period: 30m0s
+      timeout: 4h0m0s
+    labels:
+      preset-bazel-unnested: "true"
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-shared-images: "true"
+      rehearsal.allowed: "true"
+    max_concurrency: 10
+    name: pull-kubevirt-e2e-net-integration-tests
+    skip_branches:
+    - release-\d+\.\d+
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/bash
+        - -c
+        - |
+          set -ex
+          curl -L https://github.com/kubevirt/kubevirtci/pull/1193.patch | git am -3
+          curl -L https://github.com/kubevirt/kubevirt/pull/11410.patch | git am -3
+          export KUBEVIRT_PROVIDER=kind-ovn
+          make cluster-up
+          export KUBECONFIG=$(./cluster-up/kubeconfig.sh)
+          make cluster-sync
+          export PATH=$(pwd)/_out/cmd/virtctl:$PATH
+          kubectl patch kubevirt -n kubevirt kubevirt --type merge --patch '{"spec":{"configuration":{"developerConfiguration":{"featureGates":["PersistentIPs"]}}}}'
+          cd _ci-configs/kind-ovn/_ovnk/test/e2e
+          go test --test.timeout=1h --test.v --ginkgo.v --ginkgo.noColor=true --ginkgo.focus="should keep ip"
```

Comment:
nit: 
```
  --ginkgo.noColor is deprecated, use --ginkgo.no-color instead
```

Reply:
yep :) you are fast, waited for the lane to finish first